src/cloud/live/sens-data-over.md

```diff
@@ -6,202 +6,132 @@ functional_areas:
+```
 
-Any data that exports to the file becomes locked. The corresponding field in the Magento Admin becomes read-only. This ensures consistent configurations as you push the file across all environments. And every time you run this command, any new configurations are appended to your config.php file. If you need to modify or delete an existing configuration, you must edit the file manually.
+### Configuration data
 
-By using the `scd-dump` command, you can configure only the settings you want copied across all environments. After you merge the code, you can configure additional settings in Staging and Production. For sensitive configurations, you can also add those settings to environment variables. For example, you may want to add different PayPal merchant account credentials for Staging (sandbox) and Production (live).
+System settings refer to the configurations in the Magento Admin **Stores** > **Settings** > **Configuration** section.
 
-If sensitive data is found in your configurations, it is generated as environment variables to `env.php`. This file remains in the environment and should not be added to your Git environment.
+The `app/etc/config.php` file includes system configuration settings for stores, websites, modules or extensions, static file optimization, and system values related to static content deployment.
 
-### Switching between commands {#commands}
+The `config.php` file includes the following settings and configuration values:
 
-Can you switch between using `php vendor/bin/m2-ece-scd-dump` and `php ./vendor/bin/ece-tools config:dump`? Only in one specific way.
+-  Configured values for settings entered through the Magento Admin store configuration
+-  Extension list
+-  Scopes value for static content deployment (the default SCD strategy is [quick][])
 
-In 2.2 and later, you cannot delete the `config.php` file to regenerate it. You can only edit the file to change or remove a value. This file includes more information than just configuration values.
+{:.bs-callout-warning}
+The `ece-tools config:dump` command does not retrieve detailed configurations for modules, such as B2B. If you need a comprehensive configuration dump, use the [Magento `app:config:dump` command][commerce-dump], but be aware that this command may lock values in a read-only state. This affects Fastly and other important modules.
 
-If you use `php vendor/bin/m2-ece-scd-dump` to generate the file, you can switch to `php ./vendor/bin/ece-tools config:dump`. Once switched, you cannot switch back. Using the command will append the file with all configured values (default and modified) not currently captured in the file. Any modified values must be entered by editing the file.
+{:.bs-callout-info}
+Because {{site.data.var.ece}} supports only the Magento production and maintenance modes, the **Advanced** > **Developer** section is not accessible in the Admin panel. You can configure these settings using [environment variables][env-yaml].
```

Comment:
```suggestion
Because {{site.data.var.ece}} supports only the Magento production and maintenance modes, the **Advanced** > **Developer** section is not accessible in the Admin panel. You can configure these developer settings using [environment variables][env-yaml].
```